examples/with-javascript/src/config.json

```diff
@@ -0,0 +1,5 @@
+{
+    "discord_token": "",
+    "prefix": "",
+    "owners": []
```

Comment:
@Favna Owners doesn't really matter right, since the precondition isn't in the javascript example